api/src/main/java/org/openmrs/obs/handler/ImageHandler.java

```diff
@@ -94,90 +97,96 @@ public Obs getObs(Obs obs, String view) {
 				} else {
 					log.warn("MIME type of " + file.getAbsolutePath() + " is not known");
 				}
-			}
-			catch (FileNotFoundException e) {
+			} catch (FileNotFoundException e) {
 				log.error("Image " + file.getAbsolutePath() + " was not found", e);
-			}
-			catch (IOException e) {
```

Comment:
Thanks for working on this, @jwnasambu , did you intend to include line 98 and 100 on the same line